src/Learning/KWData/KWAttribute.cpp

```diff
@@ -532,22 +544,32 @@ void KWAttribute::Write(ostream& ost) const
 	}
 	ost << "\t";
 
-	// Commentaire
+	// Libelle
 	if (GetLabel() != "")
 		ost << "// " << GetLabel();
 }
 
 void KWAttribute::WriteJSONFields(JSONFile* fJSON)
 {
 	ALString sOutputString;
+	int i;
 
 	// Nom
 	fJSON->WriteKeyString("name", GetName());
 
-	// Commentaire
+	// Libelle
 	if (GetLabel() != "")
 		fJSON->WriteKeyString("label", GetLabel());
 
+	// Commentaires
+	if (GetComments()->GetSize() > 0)
```

Comment:
I wonder whether it wouldn't be more consistent to keep the contents of the `if` block in all cases, that is, have  `comments` array even when there is no comment (i.e. `GetComments()->GetSize() == 0` - in this case the array would just be empty, because the `for` loop would not have what to iterate on).